src/utils/tableSort.test.ts

```diff
@@ -0,0 +1,168 @@
+import { HeaderGroup } from 'react-table';
+import { DateTime } from 'luxon';
+
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon,
+  sortColumnProps,
+  sortColumnValues
+} from './tableSort';
+
+describe('tableSortUtil', () => {
+  describe('getColumnSortStatus', () => {
+    it('return text - ascending', () => {
+      const column = {
+        isSorted: true,
+        isSortedDesc: false
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('ascending');
+    });
+
+    it('return text - descending', () => {
+      const column = {
+        isSorted: true,
+        isSortedDesc: true
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('descending');
+    });
+
+    it('return text - none', () => {
+      const column = {
+        isSorted: false,
+        isSortedDesc: true
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('none');
+    });
+  });
+
+  describe('getHeaderSortIcon', () => {
+    it('return default caret class', () => {
+      const column = {
+        isSorted: false,
+        isSortedDesc: false
+      } as HeaderGroup;
+      expect(getHeaderSortIcon(column)).toEqual(
+        'margin-left-1 fa fa-sort caret position-absolute'
+      );
+    });
+
+    it('return down caret class', () => {
+      const column = {
+        isSorted: true,
+        isSortedDesc: true
+      } as HeaderGroup;
+      expect(getHeaderSortIcon(column)).toEqual(
+        'margin-left-1 fa fa-caret-down position-absolute'
+      );
+    });
+
+    it('return down caret class', () => {
```

Comment:
Should this test name be "return **up** caret class"?

Reply:
Hehe, yes, thanks.